rules/solid/tests/Rector/Foreach_/ChangeNestedForeachIfsToEarlyContinueRector/Fixture/multi_exprs_with_OR_both_truthy.php.inc

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\Foreach_\ChangeNestedForeachIfsToEarlyContinueRector\Fixture;
+
+use Rector\Core\Testing\Contract\RunnableInterface;
+
+class MultiExprsWithOrBothTruthy implements RunnableInterface
+{
+    public function run()
+    {
+        $partPackageList = [ new \StdClass(), 'string' ];
+        $payload = [];
+
+        foreach( $partPackageList as $partPackage )
+        {
+            if(true)
+            {
+                if (1 === 1 || 2 === 2)
+                {
+                    $payload[] = $partPackage;
+                }
+            }
+        }
+
+        return json_encode($payload);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\SOLID\Tests\Rector\Foreach_\ChangeNestedForeachIfsToEarlyContinueRector\Fixture;
+
+use Rector\Core\Testing\Contract\RunnableInterface;
+
+class MultiExprsWithOrBothTruthy implements RunnableInterface
+{
+    public function run()
+    {
+        $partPackageList = [ new \StdClass(), 'string' ];
+        $payload = [];
+
+        foreach ($partPackageList as $partPackage) {
+            if (!true) {
+                continue;
+            }
+            if (1 === 1 && 2 === 2) {
+                continue;
+            }
```

Comment:
What is real expected change here? How the correct code should look like?

Reply:
```
            if (!
                (1 === 1 || 2 === 2)
            ) {
                continue;
            }

```

Reply:
Could you update the file with that? So the fixture is correct

Reply:
Done